share/picongpu/examples/LaserWakefield/lib/python/picongpu/ui_params.py

```diff
@@ -0,0 +1,36 @@
+"""
+This file is part of PIConGPU.
+
+Copyright 2017 PIConGPU contributors
+Authors: Sebastian Starke, Jeffrey Kelling
+License: GPLv3+
+"""
+
+from picongpu.input.parameters import *
```

Comment:
you can just do explicitly: `from picongpu.input.parameters import LogScaledParameter, LinearScaledParameter`